ui/src/interfaces/INotification.ts

```diff
@@ -0,0 +1,15 @@
+import { IContainer } from "@/interfaces/IContainer";
+import { IDevice } from "@/interfaces/IDevice";
+
+export enum NotificationType {
+  DEVICE = "device",
+  CONTAINER = "container"
+}
+
+type NotificationData = IDevice | IContainer;
+
+export interface INotification {
+  id: string | number;
```

Comment:
Why don't we restrict it to a string only? Is there any valid use case for a number here?